config/rbac/role.yaml

```diff
@@ -34,6 +34,23 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+    - ""
+  resources:
+    - namespaces
+  verbs:
+    - get
```

Comment:
where do we use this?

Reply:
we need the kube-system namespace for uid..

Reply:
I don't see `kube-system` in this PR.
use it or lose it

Reply:
ok, I see it's needed for cluster role creation

Reply:
need perm to grant them